### ISSUE TITLE
Missing logs in many instances 

### ISSUE THREAD

**Issue by harmangk2** [EXL/SynergyCalls#1]
On instances where the internet is not available during the calls, the post request is not received on the server. Can the call which fails be stored locally and then reattempted on internet connection?

**Comment by EXL** [comment]
Yes, theoretically this can be done without any problems by upgrading this program. But in practice, this application has long been abandoned and outdated.
